libs/etterlatte-token-model/src/main/kotlin/no/nav/etterlatte/token/Bruker.kt

```diff
@@ -2,9 +2,8 @@ package no.nav.etterlatte.token
 
 sealed class Bruker {
     abstract fun ident(): String
-
     abstract fun erSystembruker(): Boolean
-
+    abstract fun erSaksbehandler(): Boolean
```

Comment:
Tenkte du å ta denne i bruk nå eller senere?

Jeg tenker det ofte er like ryddig å bare sjekke på typen forsåvidt (`bruker is Saksbehandler`)

Reply:
Vi kan kun bruke `bruker is System/Saksbehandler` hvis jeg har deklarert en variabel med ´Bruker` typens lik at den implisitte `get()` er kalt.

Reply:
tldr https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/853/commits/b3da18839c1ce9032471cdbe798700861cb8557d